rooms/models.py

```diff
@@ -0,0 +1,119 @@
+from django.db import models
+
+from users.models import User
+
+class Info(models.Model):
+    name = models.CharField(max_length=45)
+
+    class Meta:
+        db_table = 'infos'
+
+class Category(models.Model):
+    name = models.CharField(max_length=45)
+    
+    class Meta:
+        db_table = 'categories'
+
+class RoomType(models.Model):
+    name = models.CharField(max_length=45)
+    
+    class Meta:
+        db_table = 'room_types'
+
+class Evaluate(models.Model):
+    name = models.CharField(max_length=45)
+    
+    class Meta:
+        db_table = 'evaluates'
+
+class Convenience(models.Model):
+    name = models.CharField(max_length=45)
+    
+    class Meta:
+        db_table = 'conveniences'
+
+class Room(models.Model):
```

Comment:
`Room`과 `RoomInfo`은 뭔가 데이터가 겹치는 것처럼 느껴지는데, `Accomdation`, `Rooms`가 어떤지 의견 여쭙고 싶습니다.

그리고, 욕실과 침실은 숙소 안에 방에 해당하는 데이터라서 괜찮은데 침대같은 경우에는 방 개념이 아니라 방에 속해있는 데이터이므로 `infos` 테이블에서 관리하는게 맞지 않는 것 같아요.

**결론적으로, 아래와 같은 방향으로 db를 수정하는게 어떨지 의견 여쭤보고 싶어요!**
`Accomdation` : 숙소
`Room` : 숙소 안에 방
`RoomType` : 방의 유형 (침실, 욕실, 서재 등..)
`Bed` : 킹, 퀸 (말씀해주신대로 간단하게 하나만 정하셔도 되구요!)
`RoomBed` : 침실에 침대 2개, 서재에 침대 1개 등

![image](https://user-images.githubusercontent.com/53214510/123022782-41e52180-d411-11eb-80d4-2754b7dc24f4.png)

너무 간단하게 하려다보니 데이터가 제대로 관리가 안될 것 같아서 위와 같이 수정하면 좋을 것 같은데 팀 의견은 어떠신가요?




```suggestion
class Room(models.Model):
```

Reply:
의견 감사합니다.
저희 팀에서 분석을 하였을때는 에어비엔비의 경우, 숙소->방의 개념으로 예약을 하는것이 아닌, 방을 바로 예약하는 것으로 생각을 하였습니다.
그리고 호스트의 경우에도 숙소의 개념인 호텔을 등록 하고 방을 등록 하는것이 아닌 최종 목적지인 방을 바로 등록하는 것으로 확인을 하여, 현재 숙소와 방의 개념을 나누어서 의견 주신건 감사하지만, 팀원들과 회의 결과 숙소의 개념은 가져가지 않기로 하였습니다.

ex)
aa호텔 -> 301호 예약의 절차가 아닌 그냥 바로 301호를 예약

Reply:
자세한 설명 감사합니다~ 👍
집 전체를 빌리는 경우는 제외한다고 생각하면 되겠죠?

Reply:
넵